designs/administrator/4.4/quay-imagemanifestvuln-list/index.md

```diff
@@ -0,0 +1,29 @@
+---
+parent: Administrator
+version: 4.4
+---
+
+# Quay Image Manifest Vulnerabilities List
+
+Users that have installed the Container Security Operator now have direct access to known container image vulnerabilities in their cluster via the new **Image Manifest Vulnerabilities** nav item in the Administration section. This nav item lists the vulnerabilities on the cluster and each can be viewed to see specific details and the pods that are affected by it. Users can also access this list filtered by pod from a pod's actions menu.
+
+## Image Manifest Vulnerabilities List and Details
+
+![ImageManifestVulnerabilities List](img/1imvlist.png)
+- A new nav item  **Image Manifest Vulnerabilities**  is added to the Administration section.
+- An brief overview of the vulnerability is viewable in the list view with a link to the Quay instance the image is hosted on (for example, Quay.io.)
+
+![ImageManifestVulnerabilities Details Overview](img/2imvoverview.png)
+- Opening an Image Manifest Vulnerability shows additional details, including the list of Vulnerabilities and their severity.
```

Comment:
Should the title of the page here be "Image Manifest Vulnerability Overview" if we're looking at a single "IMV" resource?

Reply:
This is a good point. Will confirm and change, if so. Thanks.

Reply:
Turned out this is many vulnerabilities for one image, so still plural.